about/teams.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Our Teams
+icon: account-group
+---
+
+### Computational Biology Core
+Leadership: [Ashok Ragavendran](https://directory.brown.edu/uuid/60802d0b-8f18-499a-a1f3-5f9b24c390e4)
+>The COBRE-CBHD Computational Biology Core (CBC) integrates across multiple research focused Centers supporting data-intensive research using high-throughput DNA/RNA sequencing datasets. CBC provides help for experimental design and data processing pipelines for high-throughput datasets generated in individual projects, particularly for DNA/RNA sequencing data.
+
+### High-Performance Computing
+Leadership: [Ashok Ragavendran](https://directory.brown.edu/uuid/60802d0b-8f18-499a-a1f3-5f9b24c390e4)
+>Our software engineers help researchers to get the most out of high-performance computing (HPC) resources. To achieve this, we partner with users to design, debug, and optimize their code to run in an HPC environment. We also collaborate with new and non-traditional HPC users to help them to scale up and accelerate their science.
+
+### GSDC (Graphics, Software and Data Core)
```

Comment:
this should also have sections for hpc system and research technical services

Reply:
absolutely! I'm getting some blurbs from Mete, but I'll merge this first